tools/oesign/oesign.c

```diff
@@ -488,7 +512,7 @@ void _merge_config_file_options(
             &options->extended_product_id.value,
             sizeof(options->extended_product_id.value));
 
-    if (options->family_id.has_value || options->extended_product_id.has_value)
+    if (options->enable_kss.value)
```

Comment:
Also unset the flag if EnableKSS is not specified. Isn't this tested by the prop_kss.c + sign.conf combination?

Reply:
yes the test catches it. Forgot committing the change. Pushing now.